src/App.js

```diff
@@ -1,5 +1,7 @@
 //; -*- mode: rjsx;-*-
 /** @jsx jsx */
+import PropTypes from 'prop-types';
```

Comment:
What is this addition all about? I read about the package and do understand what it's for, but I'm not sure why this was added in this PR. Was it linted by ESLint? But if so, why only for `App` and `ConnectedRoute`?

Reply:
I've added this since the ESLint complains about this by default but then noticed that I did not lint sub-directories where I would have needed to fix this at a lot of places. That's why my pragmatic solution was to deactivate this rule for now:

https://github.com/elan-ev/opencast-studio/blob/88d4b40dd328ec16d4d36b29ef8c4f243b47e4b8/.eslintrc.json#L18

I still think that it makes sense to add this, so I didn't remove it from `App.js`. But I can remove it if you want me to.

Reply:
Yeah no, it's fine. Just wanted to know :)